app-center-webapps/src/main/webapp/vueLocales/locale_en.json

```diff
@@ -37,9 +37,8 @@
   "appCenter.adminSetupList.actions": "Actions",
   "appCenter.adminSetupList.remove": "Remove",
   "appCenter.adminSetupList.search": "Search",
-  "appCenter.userSetup.appDirectory": "Applications directory",
-  "appCenter.userSetup.authorized.access": "Access",
-  "appCenter.userSetup.authorized.favorite": "Favorite",
+  "appCenter.userSetup.appCenter": "Application center",
+  "appCenter.userSetup.authorized.open": "OPEN",
```

Comment:
Why this label is in capital letter ?
Perhaps we should change it to normal, and capitalize it by css

Reply:
Okay will fix it